assets/shared-components/doubleHeading.jsx

```diff
@@ -0,0 +1,37 @@
+// ----- Imports ----- //
+
+import React from 'react';
+
+
+// ----- Component ----- //
+
+export default function DoubleHeading(props) {
```

Comment:
Not specific to this line, but are we planning to introduce flow before we write too much more un-typed js?